### PR TITLE
[RLlib] Fix broken link-checker (due to final removal of deprecated py-file).

### PR DESCRIPTION
**Pull request by sven1977**
Fix broken link-checker (due to final removal of deprecated py-file).

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/rllib/rllib-env.rst [hunk 1]
```diff
@@ -594,4 +594,8 @@ For the best performance, we recommend using ``inference_mode="local"`` when pos
 Advanced Integrations
 ---------------------
 
-For more complex / high-performance environment integrations, you can instead extend the low-level `BaseEnv <https://github.com/ray-project/ray/blob/master/rllib/env/base_env.py>`__ class. This low-level API models multiple agents executing asynchronously in multiple environments. A call to ``BaseEnv:poll()`` returns observations from ready agents keyed by 1) their environment, then 2) agent ids. Actions for those agents are sent back via ``BaseEnv:send_actions()``. BaseEnv is used to implement all the other env types in RLlib, so it offers a superset of their functionality. For example, ``BaseEnv`` is used to implement dynamic batching of observations for inference over `multiple simulator actors <https://github.com/ray-project/ray/blob/master/rllib/env/remote_vector_env.py>`__.
+For more complex / high-performance environment integrations, you can instead extend the low-level
+`BaseEnv <https://github.com/ray-project/ray/blob/master/rllib/env/base_env.py>`__ class.
```
Comment (simonsays1980) [inline doc/source/rllib/rllib-env.rst:598]: But `BaseEnv` will be deprecated?

**Review by simonsays1980 — APPROVED** [review]
LGTM.

**Comment by GeneDer** [comment]
@sven1977 Not sure if it's related, I also noticed readthedocs complains about "exception: no module named ray.rllib.Policy"

**Comment by sven1977** [comment]
> @sven1977 Not sure if it's related, I also noticed readthedocs complains about "exception: no module named ray.rllib.Policy"

Yeah, this is unrelated. I'm talking to @angelinalg and @peytondmurray about this rn ...

File: doc/source/rllib/rllib-env.rst (hunk 1 above)
Comment (sven1977) [reply doc/source/rllib/rllib-env.rst:598]: Yes, at some point :)
This PR was about the renamed py-file `remote_vector_env.py`, which had been renamed a while back into `remote_base_env.py`.

**Pull request merged**